<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScorePage</class>
 <widget class="QDialog" name="ScorePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>723</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Agency FB</family>
    <pointsize>36</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Retart</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>520</y>
     <width>211</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>-10</y>
     <width>351</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>60</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff00ff;&quot;&gt;Score:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="scorenum">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>-10</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff55ff;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="superdrum_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum12.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1031</width>
     <height>732</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/taiko_cal_0801_01.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="superdrum">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum12.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="superdrum_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum12.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="superdrum_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum12.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="superdrum_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum12.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="superdrum_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum12.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="gooddrum_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="gooddrum_8">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="gooddrum_9">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="gooddrum_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="gooddrum_11">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="gooddrum_12">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum2.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="baddrum_13">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum8.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="baddrum_14">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum8.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="baddrum_15">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum8.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="baddrum_16">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum8.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="baddrum_17">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum8.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="baddrum_18">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pic.qrc">:/picture/reddrum8.jpg</pixmap>
   </property>
  </widget>
  <zorder>label_4</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label</zorder>
  <zorder>scorenum</zorder>
  <zorder>superdrum_6</zorder>
  <zorder>superdrum</zorder>
  <zorder>superdrum_2</zorder>
  <zorder>superdrum_4</zorder>
  <zorder>superdrum_5</zorder>
  <zorder>superdrum_3</zorder>
  <zorder>gooddrum_7</zorder>
  <zorder>gooddrum_8</zorder>
  <zorder>gooddrum_9</zorder>
  <zorder>gooddrum_10</zorder>
  <zorder>gooddrum_11</zorder>
  <zorder>gooddrum_12</zorder>
  <zorder>baddrum_13</zorder>
  <zorder>baddrum_14</zorder>
  <zorder>baddrum_15</zorder>
  <zorder>baddrum_16</zorder>
  <zorder>baddrum_17</zorder>
  <zorder>baddrum_18</zorder>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
